tools/inference.py

```diff
@@ -108,11 +134,17 @@ def infer() -> None:
     # Show or save the output image, depending on what's provided as
     # the command line argument.
     output = cv2.cvtColor(output, cv2.COLOR_RGB2BGR)
```

Comment:
Maybe we should also show the segmentation predictions here, but I guess we could address it separately

Reply:
I didn't understand this. Aren't we overlaying anomaly masks on line 127?

Reply:
This is what I get when I run the inferencer. It just shows the heatmap and the confidence score, no segmentation predictions.
![Screenshot from 2022-04-11 12-37-36](https://user-images.githubusercontent.com/26067904/162723555-2b84aeca-b41f-4a23-83c6-6101f15377ac.png)